### DISCUSSION TITLE
FileUpload doesn't work

### DISCUSSION THREAD

**Discussion by jesus9314** [filamentphp/filament#12703] [Help]
### Package

Form builder

### Package Version

3.2

### How can we help you?

Hello, I am doing a project with **filament v3** and when I work on my PC, I don't have any problems, but I have cloned my repository to my laptop, and the **Fileuploaders** does not work, they load to upload the files, but an 'x' appears and says error, I have configured the storage :link, I am working with laragon just like on my pc, I have set the url just like on my pc, but it doesn't work in any way.

This is my repo, thanks for your help.

[https://github.com/jesus9314/inar-technologies](url)

**Comment by leandrocfe** [comment]
What is the error?

**Reply by jesus9314** [reply]
![error fileuploader](https://github.com/filamentphp/filament/assets/45642517/2503db6f-d2f7-48dd-bca4-4365899d67e6)

This happends when I try to upload a file in any fileuploader, and i get that error in the console.

**Reply by leandrocfe** [reply]
inspect the laravel.log

**Reply by jesus9314** [reply]
![image](https://github.com/filamentphp/filament/assets/45642517/662b1414-67d5-44d5-b5c0-51f2edaf95db)
